### PR TITLE
Rebuild MODEL_ZOO

### PR DESCRIPTION
**Pull request by rentainhe**
## TODO
- [x] Rebuild model_zoo
- [x] Delete useless file
- [x] Update README
- [x] Update ChangeLog in Docs
- [x] Test all the model with pretrained weight before oneflow 0.7.0 https://github.com/Oneflow-Inc/OneTeam/issues/972#issuecomment-1022800032

## ChangeLog
- [x]  Fix pretrained weight in `vit_small_patch32_384`, `res2net50_48w_2s`
- [x] Add `resmlp_12_224_dino` and `resmlp_24_224_dino` model and pretrained weight
- [x] Rebuild a cleaner model_zoo for flowvision

## OneFlow bug
bug code: 
```python
import oneflow as flow
x = flow.randn(1, 3)
(x == 0).any()
```
fix PR: https://github.com/Oneflow-Inc/oneflow/pull/7394

### CODE REVIEW

**Comment by kaijieshi7** [comment]
确定全部重跑吗？

**Comment by rentainhe** [comment]
> 确定全部重跑吗？

V0.1.0发布前重新跑一下，之后就不用再跑了

File: results/results_imagenet.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+## Flowvision Model Zoo
+
+| model | top1 | top1_err | top5 | top5_err | parameters | img_size | crop_pct | interpolation |
+|:------|:-----|:---------|:-----|:---------|:-----------|:---------|:---------|:--------------|
+| alexnet | 56.554 | 43.446 | 79.082 | 20.918 | 61.1M  | 224 | 0.875 | bilinear |
+| vgg11   | 69.040 | 30.960 | 88.634 | 11.366 | 132.9M | 224 | 0.875 | bilinear |
+| vgg13   | 69.940 | 30.060 | 89.256 | 10.742 | 133.1M | 224 | 0.875 | bilinear |
+| vgg16   | 71.586 | 28.414 | 90.390 | 9.610  | 138.4M | 224 | 0.875 | bilinear |
+| vgg19   | 72.394 | 27.606 | 90.886 | 9.114  | 143.7M | 224 | 0.875 | bilinear |
+| vgg11_bn   | 70.378 | 29.622 | 89.814 | 10.186  | 132.9M | 224 | 0.875 | bilinear |
```
Comment (kaijieshi7) [inline results/results_imagenet.md:10]: 文档里面可以不用下划线，感觉不够美观。

File: flowvision/models/res_mlp.py [hunk 2]
```diff
@@ -461,12 +496,12 @@ def resmlpB_24_in22k(pretrained=False, progress=True, **kwargs):
...
     )
     return _create_resmlp(
-        "resmlpB_24_in22k", pretrained=pretrained, progress=progress, **model_kwargs
+        "resmlp_big_24_224_in22k_to_1k", pretrained=pretrained, progress=progress, **model_kwargs
     )
 
 
 @ModelCreator.register_model
-def resmlpB_24_dist(pretrained=False, progress=True, **kwargs):
+def resmlp_big_24_distilled_224(pretrained=False, progress=True, **kwargs):
```
Comment (kaijieshi7) [inline flowvision/models/res_mlp.py:516]: 蒸馏和原版应该只是权重不同，结构是相同的，那么这里是不是重复创建了两个一样的模型架构

**Review by kaijieshi7 — APPROVED** [review]

**Pull request merged**